lock/src/main/java/com/auth0/android/lock/provider/OAuth2WebAuthProvider.java

```diff
@@ -137,31 +139,44 @@ private void startAuthorization(Activity activity, Uri authorizeUri, String conn
     @Override
     public boolean authorize(Activity activity, @NonNull AuthorizeResult data) {
         if (!data.isValid(OAUTH2_REQUEST_CODE)) {
+            Log.w(TAG, "Invalid request.");
             return false;
         }
 
         final Map<String, String> values = helper.getValuesFromUri(data.getIntent().getData());
+        if (values.isEmpty()) {
+            Log.w(TAG, "Missing values from the response.");
```

Comment:
What is `values`? remember this can be used by users of the lib so we need to provide something meaningful